draft-packages/modal/docs/InputEditModal.stories.tsx

```diff
@@ -49,6 +48,11 @@ export default {
   title: `${CATEGORIES.components}/Modal/Input Edit Modal`,
   component: InputEditModal,
   parameters: {
+    chromatic: {
+      disable: false,
```

Comment:
Was considering making a constant for `chromatic: { disable: false }`, but then there's these situations where you'd want multiple properties, so decided to park it for now. We can refactor and optimise later :)

Reply:
I think that's fine for now, and there'll be a lot less repetition when we update tot he storywrapper design sheets anyhow. Can't wait to see all those red removals 🤤